src/RMGGeneratorDecay0.cc

```diff
@@ -43,4 +50,75 @@ void RMGGeneratorDecay0::GeneratePrimaries(G4Event* event) {
   fDecay0G4Generator->GeneratePrimaries(event);
 }
 
+void RMGGeneratorDecay0::SetMode(std::string mode) {
+  try {
+    fDecayMode = RMGTools::ToEnum<RMGGeneratorDecay0::DecayMode>(mode, "decay mode");
+  } catch (const std::bad_cast&) { return; }
+  if (fDecayMode == DecayMode::k2vbb) {
+    nuclide = "Ge76";
+    dbd_mode = 4;
+    dbd_level = 0;
+  } else if (fDecayMode == DecayMode::k0vbb) {
+    nuclide = "Ge76";
+    dbd_mode = 1;
+    dbd_level = 0;
+  } else {
+    RMGLog::Out(RMGLog::error, "Unknown decay mode");
+  }
+
+  seed = CLHEP::HepRandom::getTheSeed();
+
+  // Sanity check
+  if (seed <= 0) {
+    RMGLog::Out(RMGLog::error, "Seed seems invalid or uninitialized. Setting seed to 0.");
```

Comment:
```suggestion
    RMGLog::Out(RMGLog::error, "CLHEP seed seems invalid or uninitialized. Setting BxDecay0 seed to 0.");
```

this should be more clear, you do not set the global seed here.

Reply:
You are right, i did actually already change it locally for the warning message :-)